### ISSUE TITLE
`This branch is 86 commits ahead of master.`

### ISSUE THREAD

**Issue by nok** [openml/openml-python#599]
When will the `develop` branch be merged into the `master` branch?
https://github.com/openml/openml-python/compare/develop

It's difficult to contribute because the `master` is broken:
https://travis-ci.org/openml/openml-python/branches

**Comment by janvanrijn** [comment]
Thanks for this message. 

Contributions should be done in develop. We merge develop twice per year into master (in 2018 actually only once..) See here out git flow model: https://nvie.com/posts/a-successful-git-branching-model/ (applicable to openml-python)

**Comment by nok** [comment]
TY @janvanrijn 👍

**Comment by mfeurer** [comment]
It's on my todo list now that the tests work again.
